<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Control Application</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>372</width>
      <height>345</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="bttnnfov">
       <property name="text">
        <string>NFOV</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="bttnNormalImg">
       <property name="text">
        <string>Normal</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="bttnZoom2">
       <property name="text">
        <string>Zoom 2X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="bttnDDEOn">
       <property name="text">
        <string>DDE On</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="bttnmfov">
       <property name="text">
        <string>MFOV</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="bttnRotX">
       <property name="text">
        <string>Rot X-Axis</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="bttnZoom1">
       <property name="text">
        <string>Zoom 1X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="bttnDDEOff">
       <property name="text">
        <string>DDE Off</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="bttnwfov">
       <property name="text">
        <string>WFOV</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="bttnRotY">
       <property name="text">
        <string>Rot Y-Axis</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="bttnFrezImg">
       <property name="text">
        <string>Freeze Img</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="bttnDDEM0">
       <property name="text">
        <string>DDE Mode 0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="bttnfocusInc">
       <property name="text">
        <string>Focus +</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="bttnCentPoint">
       <property name="text">
        <string>Cent. Point</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="bttnActImg">
       <property name="text">
        <string>Active Img</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="bttnDDEM1">
       <property name="text">
        <string>DDE Mode 1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="bttnfocusdec">
       <property name="text">
        <string>Focus -</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="bttnAGOC0">
       <property name="text">
        <string>AGOC0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="bttnNegImg">
       <property name="text">
        <string>Negative</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="bttnDDEM2">
       <property name="text">
        <string>DDE Mode 2</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="bttnfocusAuto">
       <property name="text">
        <string>Auto Focus</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="bttnAGOC1">
       <property name="text">
        <string>AGOC1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="bttnPosImg">
       <property name="text">
        <string>Positive</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="bttnBPRMTD1">
       <property name="text">
        <string>BPR MTD 1</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="bttnBSNUC">
       <property name="text">
        <string>BSNUC</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="bttnMGOC">
       <property name="text">
        <string>MGOC</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="bttnAimOn">
       <property name="text">
        <string>Aim On</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="bttnBPRMTD2">
       <property name="text">
        <string>BPR MTD 2</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="bttnBNSNUC">
       <property name="text">
        <string>BNSNUC</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="bttnSavePara">
       <property name="text">
        <string>Save Para</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="bttnAimOff">
       <property name="text">
        <string>Aim Off</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="bttnSBPT">
       <property name="text">
        <string>SBPT</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>6</y>
      <width>371</width>
      <height>103</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QPushButton" name="bttnClearTx">
       <property name="text">
        <string>Clear TX</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="bttnComClose">
       <property name="text">
        <string>Close COM</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="bttnComStart">
       <property name="text">
        <string>Open COM</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="bttnClearRx">
       <property name="text">
        <string>Clear RX</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="cBoxBaudrate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="cBoxPortName">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>ttyS0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ttyS1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ttyS2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ttyS3</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>300</y>
      <width>196</width>
      <height>75</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>Manual Gain</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QPushButton" name="bttnMGainInc">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="bttnMGainDec">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="bttnMGainReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>388</x>
      <y>380</y>
      <width>196</width>
      <height>75</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>Manual Offset</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QPushButton" name="bttnMOffsetInc">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="bttnMOffsetDec">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="bttnMOffsetReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>691</x>
      <y>455</y>
      <width>207</width>
      <height>129</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>392</x>
      <y>4</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tx Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rx Data</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtbxTxData">
    <property name="geometry">
     <rect>
      <x>382</x>
      <y>30</y>
      <width>199</width>
      <height>125</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtbxRxData">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>180</y>
      <width>199</width>
      <height>113</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>bttnnfov</tabstop>
  <tabstop>bttnmfov</tabstop>
  <tabstop>bttnwfov</tabstop>
  <tabstop>bttnfocusInc</tabstop>
  <tabstop>bttnfocusdec</tabstop>
  <tabstop>bttnfocusAuto</tabstop>
  <tabstop>bttnBSNUC</tabstop>
  <tabstop>bttnBNSNUC</tabstop>
  <tabstop>bttnNormalImg</tabstop>
  <tabstop>bttnZoom2</tabstop>
  <tabstop>bttnDDEOn</tabstop>
  <tabstop>bttnRotX</tabstop>
  <tabstop>bttnZoom1</tabstop>
  <tabstop>bttnDDEOff</tabstop>
  <tabstop>bttnRotY</tabstop>
  <tabstop>bttnFrezImg</tabstop>
  <tabstop>bttnDDEM0</tabstop>
  <tabstop>bttnCentPoint</tabstop>
  <tabstop>bttnActImg</tabstop>
  <tabstop>bttnDDEM1</tabstop>
  <tabstop>bttnAGOC0</tabstop>
  <tabstop>bttnNegImg</tabstop>
  <tabstop>bttnDDEM2</tabstop>
  <tabstop>bttnAGOC1</tabstop>
  <tabstop>bttnPosImg</tabstop>
  <tabstop>bttnBPRMTD1</tabstop>
  <tabstop>bttnMGOC</tabstop>
  <tabstop>bttnAimOn</tabstop>
  <tabstop>bttnBPRMTD2</tabstop>
  <tabstop>bttnSavePara</tabstop>
  <tabstop>bttnAimOff</tabstop>
  <tabstop>bttnSBPT</tabstop>
  <tabstop>bttnClearTx</tabstop>
  <tabstop>bttnComClose</tabstop>
  <tabstop>bttnComStart</tabstop>
  <tabstop>bttnClearRx</tabstop>
  <tabstop>cBoxBaudrate</tabstop>
  <tabstop>cBoxPortName</tabstop>
  <tabstop>bttnMGainInc</tabstop>
  <tabstop>bttnMGainDec</tabstop>
  <tabstop>bttnMOffsetInc</tabstop>
  <tabstop>bttnMOffsetDec</tabstop>
  <tabstop>lineEdit_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
